notebooks/vertex_matching_engine/solutions/matching_engine_query.ipynb

```diff
@@ -0,0 +1,501 @@
+{
```

Comment:
<p>What about adding that as part of the setup script for the matching engine in <code>/scripts</code> so that all the deps &amp; setup for the repo is clear and at a single place.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/230/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>